presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -118,12 +118,11 @@ public DistributedQueryRunner(
                 servers.add(worker);
             }
 
-            Map<String, String> extraCoordinatorProperties = ImmutableMap.<String, String>builder()
-                    .put("optimizer.optimize-mixed-distinct-aggregations", "true")
-                    .put("experimental.iterative-optimizer-enabled", "true")
```

Comment:
The change here is because `ImmutableMap.builder` won't allow to overwrite `experimental.iterative-optimizer-enabled`. Alternatively i could remove the setting from here, since it's default now.